assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_grading_submission.vm

```diff
@@ -234,10 +234,10 @@
 										#set ($props = $reviewResult.getContentResource().Properties)
 										<span class="reportIcon">
 										#if ($reviewResult.getReviewReport().equals("Error"))
-											<img src="/library/skin/images/error.png" alt="$reviewResult.getReviewError()" title="$reviewResult.getReviewError()">
+											<i alt="$reviewResult.getReviewError()" title="$reviewResult.getReviewError()" class="fa fa-exclamation-triangle errorStyle"></i>
```

Comment:
Stick to `<img>` or `<span>` or such; the `<i>` tag should be used strictly for text in italics